pkg/sippyserver/server.go

```diff
@@ -2066,6 +2085,13 @@ func (s *Server) Serve() {
 			Capabilities: []string{LocalDBCapability, ComponentReadinessCapability},
 			HandlerFunc:  s.jsonTriagePotentialMatchingRegressions,
 		},
+		{
+			EndpointPath: "/api/component_readiness/triages/{id}/audit",
```

Comment:
Should this be included in the `links` section of triage? I think if we're to settle on using HATEOAS, including `audit` in links would make sense

```
"links": {
  "self": "/api/component_readiness/triages/193",
  "audit": "/api/component_readiness/triages/193/audit"
}
```

WDYT?

Reply:
Ah yes, that is a very good point. There are some others that also should be included; so I will do them all together in a follow-up.